src/components/common/TmSessionImport.vue

```diff
@@ -165,9 +168,19 @@ export default {
     }
   }),
   computed: {
-    ...mapGetters([`connected`])
+    ...mapGetters([`connected`]),
+    inExtension() {
+      if (this.$store.getters.extension) {
```

Comment:
I believe this would always return true.

In addition to what Fabian said above, you need to check `extension.enabled`.

Reply:
It isn't always true and it does work in my testing. It comes back as undefined when the getter `extension` is not defined in the extension and as defined when in the website. An alternative would be checking the chrome object to see what type of window you are in.